highbury/chain.json

```diff
@@ -0,0 +1,99 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "highbury",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://www.fury.black/",
+  "pretty_name": "Highbury",
+  "chain_id": "highbury_710-1",
+  "bech32_prefix": "fury",
```

Comment:
Please register this prefix
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Registration in progress. Please find the active PR in this link:
[https://github.com/satoshilabs/slips/pull/1600](url)